systemtest/src/main/java/io/strimzi/systemtest/resources/crd/KafkaConnectResource.java

```diff
@@ -35,22 +36,22 @@
         return Crds.kafkaConnectOperation(ResourceManager.kubeClient().getClient());
     }
 
-    public static DoneableKafkaConnect kafkaConnect(String name, int kafkaConnectReplicas) {
-        return kafkaConnect(name, name, kafkaConnectReplicas);
+    public static DoneableKafkaConnect kafkaConnect(String name, int kafkaConnectReplicas, boolean allowNetworkPolicyAccess) {
+        return kafkaConnect(name, name, kafkaConnectReplicas, allowNetworkPolicyAccess);
     }
 
-    public static DoneableKafkaConnect kafkaConnect(String name, String clusterName, int kafkaConnectReplicas) {
+    public static DoneableKafkaConnect kafkaConnect(String name, String clusterName, int kafkaConnectReplicas, boolean allowNetworkPolicyAccess) {
         KafkaConnect kafkaConnect = getKafkaConnectFromYaml(PATH_TO_KAFKA_CONNECT_CONFIG);
-        return deployKafkaConnect(defaultKafkaConnect(kafkaConnect, name, clusterName, kafkaConnectReplicas).build());
+        return deployKafkaConnect(defaultKafkaConnect(kafkaConnect, name, clusterName, kafkaConnectReplicas).build(), allowNetworkPolicyAccess);
     }
 
-    public static DoneableKafkaConnect kafkaConnectWithMetrics(String name, int kafkaConnectReplicas) {
-        return kafkaConnectWithMetrics(name, name, kafkaConnectReplicas);
+    public static DoneableKafkaConnect kafkaConnectWithMetrics(String name, int kafkaConnectReplicas, boolean allowNetworkPolicyAccess) {
+        return kafkaConnectWithMetrics(name, name, kafkaConnectReplicas, allowNetworkPolicyAccess);
     }
 
-    public static DoneableKafkaConnect kafkaConnectWithMetrics(String name, String clusterName, int kafkaConnectReplicas) {
+    public static DoneableKafkaConnect kafkaConnectWithMetrics(String name, String clusterName, int kafkaConnectReplicas, boolean allowNetworkPolicyAccess) {
         KafkaConnect kafkaConnect = getKafkaConnectFromYaml(PATH_TO_KAFKA_CONNECT_METRICS_CONFIG);
```

Comment:
So as i can see basically you change all our Kafka CR methods is a way that every time when we deploy for instance KafkaConnect, it must be with the additional parameter `allowNetworkPolicyAccess`. Would it be better to overload  method and encapsulate default value of `allowNetworkPolicyAccess`? 

```
Currently -> You have only these master methods with lot of parameters.

`public static DoneableKafkaConnect kafkaConnect(String name, int kafkaConnectReplicas, boolean allowNetworkPolicyAccess)`

Goal -> Master method with auxiliary methods like 

1) `public static DoneableKafkaConnect kafkaConnect(String name, int kafkaConnectReplicas, boolean allowNetworkPolicyAccess)`
2) ``public static DoneableKafkaConnect kafkaConnect(String name, int kafkaConnectReplicas)``

where is second example you will need to decide has default value for the third argument of method in a 1). 

Does it make sense?